### PR TITLE
[voltLib] Support writing back ast as VOLT data

### PR DESCRIPTION
**Pull request by khaledhosny**
Adds `__str__()` method to ast classes that writes back VOLT data. Tries to replicate VOLT syntax idiosyncrasies as much as possible for better round-trip conversion.

### CODE REVIEW

File: Tests/voltLib/parser_test.py [hunk 1]
```diff
@@ -473,6 +473,21 @@ def test_lookup_name_starts_with_letter(self):
                 'END_SUBSTITUTION\n'
             ).statements
 
+    def test_lookup_commnets(self):
```
Comment (anthrotype) [inline Tests/voltLib/parser_test.py:476]: typo

```suggestion
    def test_lookup_comments(self):
```

File: Lib/fontTools/voltLib/ast.py [hunk 2]
```diff
@@ -83,6 +114,9 @@ def __init__(self, glyph, location=None):
     def glyphSet(self):
         return (self.glyph,)
 
+    def __str__(self):
+        return f' GLYPH "{self.glyph}"'
```
Comment (anthrotype) [inline Lib/fontTools/voltLib/ast.py:118]: just curious, why start this and others with empty space? Is it some kind of in-built indentation?

**Comment by moyogo** [comment]
LGTM.
Just wondering why you added spaces making double space.

File: Lib/fontTools/voltLib/ast.py (hunk 2 above)
Comment (khaledhosny) [reply Lib/fontTools/voltLib/ast.py:118]: That is to match VOLT spacing, there are different ways to do it but this seemed to be more consistent.

**Comment by khaledhosny** [comment]
> Just wondering why you added spaces making double space.

To match VOLT spacing. I have a punch of VTP files and if I round them through this they show no changes (except for empty `IN_CONTEXT` but I left this out because it might break backward compatibility).

**Pull request merged**